src/Web/Slack/Types/Channel.hs

```diff
@@ -22,7 +23,7 @@ data Channel = Channel { _channelId         :: ChannelId
                        , _channelMembers    :: Maybe [UserId]
                        , _channelTopic      :: Maybe Topic
                        , _channelPurpose    :: Maybe Purpose
-                       , _channelIsMember   :: Bool
+                       , _channelIsMember   :: Maybe Bool
```

Comment:
I just foresee that any piece of code which uses this field will wrap it in a `fromMaybe False`.